docs/experimental-features.md

```diff
@@ -571,20 +570,15 @@ ipfs config --json Experimental.ShardingEnabled true
 
 Experimental, default-disabled.
 
-Utilizes pubsub for publishing ipns records in real time.
+Utilizes pubsub for publishing ipns records in real-time.
```

Comment:
```suggestion
Utilizes pubsub for publishing ipns records in real time.
```
in the noun form, two words is preferred: https://www.merriam-webster.com/dictionary/real%20time